.gitignore

```diff
@@ -1,7 +1,7 @@
 *.jl.cov
 *.jl.*.cov
 *.jl.mem
-/Manifest.toml
+Manifest.toml
```

Comment:
keeping the docs manifest around is useful to avoid breaking doctests if packages we use change their string represetations